whisper_online.py

```diff
@@ -441,7 +441,20 @@ def split(self, text):
     # supported by fast-mosestokenizer
     if lan in "as bn ca cs de el en es et fi fr ga gu hi hu is it kn lt lv ml mni mr nl or pa pl pt ro ru sk sl sv ta te yue zh".split():
         from mosestokenizer import MosesTokenizer
-        return MosesTokenizer(lan)
+        # use MosesTokenizer if it has a `split` function
+        if "split" in dir(MosesTokenizer):
+            return MosesTokenizer(lan)
+
+        # fallback splitter for platforms that don't have MosesTokenizer.split such as Apple Silicon
+        class Splitter:
+            def __init__(self):
+                from mosestokenizer import MosesSentenceSplitter
+                self.splitter = MosesSentenceSplitter(lan)
+
+            def split(self, text):
+                return self.splitter(text)
```

Comment:
I think this is incorrect since MosesSentenceSplitter expects a list of strings, not a string.

can be fixed this way:

return self.moses_splitter([text])

Reply:
Good catch! Thanks!